CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+## [0.4.1] - 2023-04-06
+### Added
+- Added the capabilities of trading fungible tokens on planetmint
```

Comment:
This is a new feature and should be bumped to 0.5.0, no?

Reply:
true, I perceived it more as a bug as I expected it to be handled before and the liquid wallet does this oob. but you're right I'll increase the number